src/utilities/getCurrentDate.ts

```diff
@@ -30,8 +32,8 @@ export function generateCurrentDateObject(): DateData {
 	return generateDateObject(currentDate);
 }
 
-export function generateDateObject(day: Date): DateData {
-	const currentDate = day;
+export function generateDateObject(dayObj: Date): DateData {
```

Comment:
Changed variable name due to bug in line 39. Kept passing in "day: Date" object instead of just the day number